agent/agent.go

```diff
@@ -165,6 +172,7 @@ func (agent *Agent) connect() error {
 	agent.logger.Debugf(context.Background(), "started opamp client")
 
 	go agent.healthCheck()
+	go agent.usageReport()
```

Comment:
What is the difference between healthCheck and usageReport?

Reply:
The`healthCheck` is the one that reports Refinery health through the opamp health check capability.
`usageReport` is to report the usage metrics to through the custom message. 
I wanted to separate the usage metrics calculation from the actual reporting interval. That's why I created a separate goroutine for reporting itself